<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>config</class>
 <widget class="QDialog" name="config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuração</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #000000;
color: #E5E5E5</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>441</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Configuração necessária para a instalação das músicas</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Client ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Secret ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Refresh Token</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="client_id">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	background-color: #14213D;
	border: solid;
	border-radius: 6px;
	color: rgb(255, 255, 255);
}

QLineEdit:hover {
	background-color: #183B54;
	border: 0.5px solid rgb(153, 153, 153);
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="secret_id">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	background-color: #14213D;
	border: solid;
	border-radius: 6px;
	color: rgb(255, 255, 255);
}

QLineEdit:hover {
	background-color: #183B54;
	border: 0.5px solid rgb(153, 153, 153);
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="refresh_token">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	background-color: #14213D;
	border: solid;
	border-radius: 6px;
	color: rgb(255, 255, 255);
}

QLineEdit:hover {
	background-color: #183B54;
	border: 0.5px solid rgb(153, 153, 153);
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="send">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: solid;
	border-radius: 3px;
	background-color: #14213D;
}

QPushButton:hover {
	background-color: #183B54;
}</string>
        </property>
        <property name="text">
         <string>Enviar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border: solid;
	border-radius: 3px;
	background-color: #14213D;
}

QPushButton:hover {
	background-color: #183B54;
}</string>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
